README.md

```diff
@@ -30,281 +30,19 @@
-| Table name                       | `surrealdb.Table(name)`                                                                                     | `surrealdb.Table("users")`                                                             |
-| Record ID                        | `surrealdb.RecordID{Table: string, ID: any}`                                                                | `surrealdb.RecordID{Table: "customers", ID: 1}, surrealdb.NewRecordID("customers", 1)` |
-| Geometry Point                   | `surrealdb.GeometryPoint{Latitude: float64, Longitude: float64}`                                            | `surrealdb.GeometryPoint{Latitude: 11.11, Longitude: 22.22`                            |
-| Geometry Line                    | `surrealdb.GeometryLine{GeometricPoint1, GeometricPoint2,... }`                                             |                                                                                        |
-| Geometry Polygon                 | `surrealdb.GeometryPolygon{GeometryLine1, GeometryLine2,... }`                                              |                                                                                        |
-| Geometry Multipoint              | `surrealdb.GeometryMultiPoint{GeometryPoint1, GeometryPoint2,... }`                                         |                                                                                        |
-| Geometry MultiLine               | `surrealdb.GeometryMultiLine{GeometryLine1, GeometryLine2,... }`                                            |                                                                                        |
-| Geometry MultiPolygon            | `surrealdb.GeometryMultiPolygon{GeometryPolygon1, GeometryPolygon2,... }`                                   |                                                                                        |
-| Geometry Collection              | `surrealdb.GeometryMultiPolygon{GeometryPolygon1, GeometryLine2, GeometryPoint3, GeometryMultiPoint4,... }` |                                                                                        |
-
-## Helper Types
-
-### surrealdb.O
-
-For some methods like create, insert, update, you can pass a map instead of an struct value. An example:
-
-```go
-person, err := surrealdb.Create[Person](context.Background(), db, models.Table("persons"), map[any]any{
-	"Name":     "John",
-	"Surname":  "Doe",
-	"Location": models.NewGeometryPoint(-0.11, 22.00),
-})
-```
-
-This can be simplified to:
-
-```go
-person, err := surrealdb.Create[Person](context.Background(), db, models.Table("persons"), surrealdb.O{
-	"Name":     "John",
-	"Surname":  "Doe",
-	"Location": models.NewGeometryPoint(-0.11, 22.00),
-})
-```
-
-Where surrealdb.O is defined below. There is no special advantage in using this other than simplicity/legibility.
-
-```go
-type surrealdb.O map[any]any
-```
-
-### surrealdb.Result[T]
```

Comment:
Removed because it was inappropriate. You should use `surrealdb.RPCResponse` to receive RPC responses.